packages/fxa-shared/express/routing.ts

```diff
@@ -6,7 +6,7 @@ import express from 'express';
 import Logger from '../lib/logger';
 import cors from './cors';
 
-type RouteMethod =
+export type RouteMethod =
```

Comment:
This is exported, but not sure if anything imports/uses it (apart from the re-exported `RouteDefinition.method` below). 🤷

Reply:
Good catch, I exported this and had imported it elsewhere before realizing I needed to use a different type.